source/rest/manager/README.md

```diff
@@ -1,12 +1,14 @@
-# php-base - PHP Slim Server library for CometVisu Manager backend
+# php-base - PHP Slim 4 Server library for CometVisu Manager backend
 
 * [OpenAPI Generator](https://openapi-generator.tech)
-* [Slim Framework Documentation](https://www.slimframework.com/docs/)
+* [Slim 4 Documentation](https://www.slimframework.com/docs/v4/)
+
+This server has been generated with [Slim PSR-7](https://github.com/slimphp/Slim-Psr7) implementation.
 
 ## Requirements
 
 * Web server with URL rewriting
-* PHP 7.0 or newer
+* PHP 7.4 or newer
```

Comment:
Doesn't this need to be bumped to 8.0 now?

Reply:
Although this file cah been created by a generator, I changed the version by hand, because this file does not seem to be overriden by new generator call.